derive_client/constants.py

```diff
@@ -1,8 +1,13 @@
 """
 Constants for Lyra.
 """
+
+from pathlib import Path
+
 from derive_client.enums import Environment
 
+REPO_ROOT = Path(__file__).parent.parent
+
 PUBLIC_HEADERS = {"accept": "application/json", "content-type": "application/json"}
 
 TEST_PRIVATE_KEY = "0xc14f53ee466dd3fc5fa356897ab276acbef4f020486ec253a23b0d1c3f89d4f4"
```

Comment:
I feel like the test private key should be in the constants.

Reply:
it is tho, right?

Reply:
*test* constants